### DISCUSSION TITLE
rename questions

### DISCUSSION THREAD

**Discussion by warior456** [terrarium-earth/Ad-Astra#2] [Q&A] [answered]
when will the version with the new name id come out

**Comment by Bunny-Proxima** [comment, marked as answer]
It should be within 1 - 3 days, the id refactor is almost done we just have to fix structures.

**Reply by warior456** [reply]
alright then i will wait untill then to update my modpack (adding the mod for the first time)

**Reply by AlexNijjar** [reply]
It's out now. Easier than I though.

**Reply by warior456** [reply]
![image](https://user-images.githubusercontent.com/66562258/185491141-2024a24e-be54-4f88-8d1e-30aad9031424.png)
kinda difficult to search anything on this mod now

**Reply by Aulluna** [reply]
Yea I've seen this mod before , but It doesnt really matter , the mod is not that well known and pretty much outdated . So I dont think we should worry about it . Once the mod get popular enough and showcase videos appear , there wont be any issue

**Comment by AlexNijjar** [comment]
There's a guidebook with a "Preparing to launch" entry. It's not exactly a flight checklist but it tells you what you need to bring.
